src/runtime/tests/refiner-test.ts

```diff
@@ -17,338 +17,362 @@ import {Schema} from '../schema.js';
 import {Flags} from '../flags.js';
 
 describe('refiner', () => {
-    it('Refines data given an expression 1', Flags.withFieldRefinementsAllowed(async () => {
-        const manifestAst = parse(`
-            particle Foo
-                input: reads Something {num: Number [ (num/-3 < 0 and num*num == 36) or num == 0 ] }
-        `);
-        const typeData = {'num': 'Number'};
-        const ref = Refinement.fromAst(manifestAst[0].args[0].type.fields[0].type.refinement, typeData);
-        const data = {
-            num: 6
-        };
-        const res = ref.validateData(data);
-        assert.strictEqual(res, data.num === 0 || data.num === 6);
-    }));
-    it('Refines data given an expression 2', () => {
-      const manifestAst = parse(`
-          particle Foo
-              input: reads Something {num: Number } [(num/-3 < 0 and num*num == 36) or num == 0]
-      `);
-      const typeData = {'num': 'Number'};
-      const ref = Refinement.fromAst(manifestAst[0].args[0].type.refinement, typeData);
-      const data = {
-          num: 6
-      };
-      const res = ref.validateData(data);
-      assert.strictEqual(res, data.num === 0 || data.num === 6);
+  it('Refines data given a field refinement', Flags.withFieldRefinementsAllowed(async () => {
```

Comment:
IIRC, phrases should start with a lowercase letter. `it` is the start of the sentence.